src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,5 +1,17 @@
 package mate.academy
 
+import mate.academy.model.figure.Figure
+import mate.academy.service.FigureSupplier
+
+private const val ARRAY_SIZE = 50
+private val figureSupplier: FigureSupplier = FigureSupplier()
+
 fun main() {
-    // draw all figures here
+    val figures: Array<Figure?> = Array(ARRAY_SIZE) { null }
+    for (i in 0 until ARRAY_SIZE) {
+        figures[i] = if (i < ARRAY_SIZE / 2) figureSupplier.getRandomFigure() else figureSupplier.getDefaultFigure()
```

Comment:
Please add curve braces for long if/else conditions for improving readability and avoiding mistakes

Reply:
fixed